sdk/storage/storage-file-share/src/generated/src/operations/share.ts

```diff
@@ -449,7 +449,7 @@ const createPermissionOperationSpec: coreHttp.OperationSpec = {
       headersMapper: Mappers.ShareCreatePermissionHeaders
     }
   },
-  isXML: false,
+  isXML: true,
```

Comment:
Should this be reverted, @XiaoningLiu?
One Integration test failed because of this change.

Same diff is in the codegen from the unified swagger too.
https://github.com/Azure/azure-sdk-for-js/pull/5980/files#diff-ea248212a35437cb5ab4aee596288febL452-R448

Reply:
@jeremymeng confirms the bug, you can revert this specific change which should fix the only test failure :)

Reply:
@HarshaNalluru is this a bug from generator?

Reply:
> It's a known codegen bug. We should keep the existing one.
> Reply to [#6032 (comment)](https://github.com/Azure/azure-sdk-for-js/pull/6032#discussion_r342782823)
> Edit - Cleaning up the comment

Why previous generated code doesn't have this?

Reply:
Not sure. Adding transforms for this in the swagger/readme didn't seem trivial to me. 

Logged an issue for further work on this specific issue - https://github.com/Azure/azure-sdk-for-js/issues/6070.

@joheredi, have you seen this when you were working with file's swagger JSON?

Reply:
Yes, I talked to @jeremymeng about this, he mentioned it was a bug in autogen and suggested to manually replace to keep "false" until the fix was ready